app_build_suite/build_steps/helm.py

```diff
@@ -298,8 +298,8 @@ def run(self, config: argparse.Namespace, _: Context) -> None:
             "lint",
             config.chart_dir,
         ]
-        if config.debug:
-            args.append("--verbose")
+    
+        args.append("--verbose")
```

Comment:
Why not put it directly in the `args` list?

Reply:
make sense, done